zazu/util.py

```diff
@@ -137,3 +148,10 @@ def pprint_list(data):
 def raise_uninstalled(pkg_name):
     """Raises a exception for a missing package"""
     raise click.ClickException('{0} not found, install it via "apt-get install {0}" or "brew install {0}"'.format(pkg_name))
+
+
+def get_plugin_list(plugin_subclass):
+    """helper function to pull lists of plugins"""
```

Comment:
I believe most of the other function comments start with uppercase. Per [PEP8](https://www.python.org/dev/peps/pep-0008/#comments) I think it should be capitalized.